assignment4.js

```diff
@@ -0,0 +1,16 @@
+let files = ["./f1","./f2","./f3", "./f4", "./f5"], i = 0;
+fs = require('fs');
+function fileReader(file){
+  fs.readFile(fileList[i], 'utf8' , function (err,data) {
```

Comment:
What is `fileList`? This is a Runtime Error, did you run the code before submission.